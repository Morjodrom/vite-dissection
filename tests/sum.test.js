import {expect, test} from 'vitest';
import {sum} from 'modules/sum.js';

test('1 + 2 = 3', () => {
	expect(sum(1, 2)).toBe(3);
});

test('1 + -Infinity = -Infinity', () => {
	expect(sum(1, -Infinity)).toBe(-Infinity);
});

test('sum for snapshots', () => {
	// noinspection JSUnresolvedReference
	expect(sum(1, 2)).toMatchSnapshot();
})
test('sum for customized snapshots', () => {
	// noinspection JSUnresolvedReference
	expect(`<div>1 + 2 = 3</div>`).toMatchFileSnapshot('./custom-snapshots/sum.test.snapshot.html');
})