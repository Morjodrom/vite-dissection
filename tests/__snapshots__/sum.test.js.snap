// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`sum for snapshots 1`] = `3`;
